app/mixins/organization-route.js

```diff
@@ -0,0 +1,10 @@
+import Ember from 'ember';
+
+export default Ember.Mixin.create({
+  setupController: function(controller, model) {
+    this._super(controller, model);
+    return this.store.find('organization').then(function(organizations){
+      controller.set('organizations', organizations);
```

Comment:
still not sure that putting the async loading of organizations into this sync `setupController` hook is the best solution. The controller or template will need to do extra work to accommodate for the fact that sometimes the 'organizations' property will be in a loading state. I think it would be less surprising if the organizations were loaded by a parent route (in an async hook) that is common to all the places where you need organizations (I would do it in a shared "authenticated" route or something similar — that route would be the parent of everything the user can see/do after logging in). Then any template would be able to be 100% certain that the organizations property is what they think it is.